<?xml version="1.0"?>

<launch>
    <arg name="swing_id" default="[3, 4]"/>

    <arg name="target_dx1" default="0.3"/>
    <arg name="target_dx2" default="0.3"/>
    <arg name="target_dx3" default="0.3"/>
    <arg name="target_dx4" default="0.3"/>

    <!--arg name="target_dy" default="0.0"/-->

    <arg name="target_dy1" default="0.0"/>
    <arg name="target_dy2" default="0.0"/>
    <arg name="target_dy3" default="0.0"/>
    <arg name="target_dy4" default="0.0"/>

    <arg name="swing_t1" default="[0.0, 4.0]"/>
    <arg name="swing_t2" default="[0.0, 4.0]"/>
    <arg name="swing_t3" default="[0.0, 4.0]"/>
    <arg name="swing_t4" default="[0.0, 4.0]"/>


    <node name="roll_feet" pkg="casannis_walking" type="roll_node.py" output="screen">
        <param name="sw_id" value="$(arg swing_id)"/>
        <param name="tgt_dx1" value="$(arg target_dx1)"/>
        <param name="tgt_dx2" value="$(arg target_dx2)"/>
        <param name="tgt_dx3" value="$(arg target_dx3)"/>
        <param name="tgt_dx4" value="$(arg target_dx4)"/>

        <!--param name="target_dy" value="$(arg target_dy)"/-->
        <param name="tgt_dy1" value="$(arg target_dy1)"/>
        <param name="tgt_dy2" value="$(arg target_dy2)"/>
        <param name="tgt_dy3" value="$(arg target_dy3)"/>
        <param name="tgt_dy4" value="$(arg target_dy4)"/>

        <param name="sw_t1" value="$(arg swing_t1)"/>
        <param name="sw_t2" value="$(arg swing_t2)"/>
        <param name="sw_t3" value="$(arg swing_t3)"/>
        <param name="sw_t4" value="$(arg swing_t4)"/>

    </node>
</launch>
